core/jobs/batch_jobs/refresher_exp_id_jobs.py

```diff
@@ -0,0 +1,93 @@
+# coding: utf-8
+#
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Jobs that extract Collection models information."""
+
+from __future__ import annotations
+
+from core.domain import exp_fetchers
+from core.jobs import base_jobs
+from core.jobs.io import ndb_io
+from core.jobs.types import job_run_result
+from core.platform import models
+
+import apache_beam as beam
+
+from typing import Iterable, Tuple
+
+MYPY = False
+if MYPY: # pragma: no cover
+    from mypy_imports import exp_models
+
+
+(exp_models,) = models.Registry.import_models([models.NAMES.exploration])
+
+datastore_services = models.Registry.import_datastore_services()
+
+
+class FilterRefresherExplorationIdJob(base_jobs.JobBase):
```

Comment:
Do we also want to include the owner email?

Reply:
@vojtechjelinek we discussed that we only require exp id and state name,
but we will require it if we are sending mail so let me know

Reply:
@vojtechjelinek email and state names are now handled PTAL